db/system_keyspace.cc

```diff
@@ -2332,8 +2332,8 @@ class runtime_info_table : public memtable_filling_virtual_table {
 
 class versions_table : public memtable_filling_virtual_table {
 public:
-    explicit versions_table()
-        : memtable_filling_virtual_table(build_schema()) {
+    explicit versions_table(replica::database& db)
```

Comment:
We should avoid passing replica::database which is replica-side to a coordinator-side class.

Reply:
Virtual tables are replica side. These are later wrapped by `replica::table` objects.

Reply:
Ah, ok.